cmd/install.go

```diff
@@ -79,6 +68,37 @@ var installCommand = &cobra.Command{
 			pkg.Entrypoint = []string{customEntrypoint}
 		}
 
+		installPath := viper.GetString("install_path")
+		pm := packages.NewPackageManager(installPath)
+
+		var installed *packages.Package
+		hasInstall := pm.HasInstallation(pkg.Name)
+		if hasInstall {
+			installed, err = pm.Load(pkg.Name)
+			if !forceInstall && err != nil {
+				return fmt.Errorf("there's already an installation of %s, but there was an error loading the package, err: %s", pkg.Name, err.Error())
+			}
+
+			fmt.Printf("Looks like you already have %s installed as %s.\n", installed.Image, path.Join(installPath, pkg.Name))
+
+			if !assumeYes {
+				if equal, err := installed.HasChanges(ctx, cli); err != nil {
+					return err
+				} else if !equal {
+					if !prompter.YN("There are differences between the installed version of the package and the image.\nAre you sure you would like to overwrite these changes?", false) {
```

Comment:
could we think on inspiring from `pkg.PreinstallMessage` and provide the changes to the user for him to be able to decide wether to override or not?

Reply:
That would be a nice feature. I'll see if i can get that working.

Reply:
@tjamet Actually do you mind if I work on adding this feature in a separate pr?

Reply:
I found a package from google that provides a diff of a given struct. It's not the prettiest output but is nice to see it in the diff format.